libpages/server.go

```diff
@@ -286,7 +319,6 @@ func (s *Server) logRequest(sri *ServedRequestInfo, requestPath string) {
 		zap.Bool("invalid_config", sri.InvalidConfig),
 	)
 }
-
```

Comment:
Function separator line now missing?

Reply:
Thanks; I got it back from lost-n-found.